index.js

```diff
@@ -8,7 +8,7 @@
 
   const githubId = process.env.GITHUB_ID
   const githubToken = process.env.GITHUB_OAUTH
-  const githubOrganization = process.env.GITHUB_ORGA
+  const githubOrganization = process.argv[2]
```

Comment:
Maybe you could use the environment variable if the param is not present ?

Reply:
For the orga and stats files?

Reply:
Sure 😄